### PR TITLE
Release -config and -test-kotlin to Maven Central

### PR DESCRIPTION
**Pull request by ibauersachs**

### CODE REVIEW

File: jicoco-config/pom.xml [hunk 1]
```diff
@@ -16,50 +16,47 @@
...
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>org.jitsi</groupId>
         <artifactId>jicoco-parent</artifactId>
         <version>1.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
```
[the remark this answers is not in the document]
Comment (ibauersachs) [reply jicoco-config/pom.xml:27]: You're right, seems I had some mess with a locally installed build.

File: jicoco/src/test/java/org/jitsi/retry/RetryStrategyTest.java [hunk 2]
```diff
@@ -15,25 +15,32 @@
...
 public class RetryStrategyTest
 {
     @Test
     public void testRetryCount()
-        throws InterruptedException
     {
-        RetryStrategy retryStrategy = new RetryStrategy();
+        AtomicInteger scheduleCalls = new AtomicInteger();
+        AtomicReference<TaskRunner> runner = new AtomicReference<>();
+        ScheduledExecutorService mockedExecutor = mock(ScheduledExecutorService.class);
```
[the remark this answers is not in the document]
Comment (ibauersachs) [reply jicoco/src/test/java/org/jitsi/retry/RetryStrategyTest.java:37]: FakeScheduledExecutorService is abstract. How do I use that without implementing 14 (!) methods?

[the remark this answers is not in the document]
Comment (ibauersachs) [reply jicoco/src/test/java/org/jitsi/retry/RetryStrategyTest.java:37]: Bit of a fiddle with Mockito to get this running:
```
FakeScheduledExecutorService mockedExecutor = mock(FakeScheduledExecutorService.class,
            withSettings()
                .useConstructor()
                .defaultAnswer(CALLS_REAL_METHODS)
        );
```

[the remark this answers is not in the document]
Comment (bbaldino) [reply jicoco/src/test/java/org/jitsi/retry/RetryStrategyTest.java:37]: Ah, we use it with a `spy`, but that's a mocking framework concept (we use `mockk`)--but I don't think we have one we use for the java code, so this is probably fine.

[the remark this answers is not in the document]
Comment (ibauersachs) [reply jicoco/src/test/java/org/jitsi/retry/RetryStrategyTest.java:37]: Well, technically that spy of mockk is still a mock - someone has to implement those abstract methods. Got it to work with the above initialization. It's closer to what the original test was doing.

[the remark this answers is not in the document]
Comment (bbaldino) [reply jicoco/src/test/java/org/jitsi/retry/RetryStrategyTest.java]: I hadn't seen your second comment above when I made mine.  Sure, that looks good.

**Review by bbaldino — APPROVED** [review]

**Pull request merged**